tools/cli-help-api/main.go

```diff
@@ -92,10 +104,56 @@ func showHelpText(w io.Writer, component string) error {
 	} else if err != nil {
 		return err
 	}
+
+	src, err := os.Open("./flags.json")
```

Comment:
```suggestion
	src, err := os.Open("flags.json")
```
./ will not work on windows